quinn-proto/src/connection/mod.rs

```diff
@@ -706,10 +706,13 @@ where
                     return;
                 }
 
+                self.stats.packet_rx.packets += 1;
```

Comment:
I now actually recognized the name `packet` is a bit ambiguous in the Quic context, due to the possibility of sending multiple Quic "packets" inside one UDP datagrams/packet. What this really counts is UDP datagrams.

But I didn't want to be inconsistent => Either all should be names packets, or get renamed to datagrams (or something else)

Reply:
Hmm, good catch. `datagrams` could also be confused with datagram frames. Would `udp_datagrams` be too verbose?

Reply:
It's fine to me. But I think @djc might have stronger opinions on verbosity :-) 

quic datagram frames  would be counted as part of `FrameStats`.

Reply:
> quic datagram frames would be counted as part of FrameStats.

Yeah, so in principle it's not ambiguous, I'm just trying to minimize risk of confusion. I think if we rename the struct to `UdpStats` and `udp_tx` then if we call it `datagrams` it might be sufficiently clear in practice, especially with a good doc comment.

Reply:
I went with that now. Sounds good to me.